src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerEditingTarget.java

```diff
@@ -265,7 +265,20 @@ public void execute(ProfileOperationResponse response)
                @Override
                public void execute()
                {
-                  commands_.closeSourceDoc().execute();
+                  server_.clearProfile(getPath(), new ServerRequestCallback<JavaScriptObject>()
+                  {
+                     @Override
+                     public void onResponseReceived(JavaScriptObject response)
+                     {
+                        commands_.closeSourceDoc().execute();
+                     }
+                     
+                     @Override
+                     public void onError(ServerError error)
+                     {
+                        Debug.logError(error);
```

Comment:
Yes. I think we should either, show the error and keep the document open or silently close since the only side-effect is to have files in a temp cache. Changed.